pydantic/main.py

```diff
@@ -284,6 +303,8 @@ def __init__(__pydantic_self__, **data: Any) -> None:  # type: ignore
         __tracebackhide__ = True
         __pydantic_self__.__pydantic_validator__.validate_python(data, self_instance=__pydantic_self__)
 
+    __init__.__pydantic_base_init__ = True  # type: ignore
```

Comment:
Custom init indicator. This way we do not need comparing an init method to `BaseModel.__init__`. This allows to avoid marking `RootModel.__init__` as custom init.